pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json

```diff
@@ -270,6 +270,23 @@
     "any",
     "base"
   ],
+  "arch": [
```

Comment:
I'd call this `archlinux` just so it's not ambiguous to people that don't know it's a Linux distro.  Otherwise, seeing things like `arch-x64` will be pretty confusing (are we talking about "x64 architecture"?)

Reply:
I see what you are saying, but that is going to be a muchmore involved fix. As it is, ID from /etc/os-release says `arch` so most RID computation in dotnet computes that to `arch-x64`. Fixing everything that computes the RID is going to be tricky and special-casing Arch Linux. Should I go down that route?

Reply:
@ericstj suggestions?

Reply:
Interesting, this is the first time we've looked at a linux distro name that collides with another concept in the RID graph (let's hope someone doesn't come up with Arm linux).  

RIDs aren't exactly supposed to be user friendly, so I don't think its a blocker, but I do agree this is confusing.  WWAPMD? (what would a PM do?)  @terrajobst @richlander